daml-script/test/daml/TestInterfaces.daml

```diff
@@ -5,6 +5,7 @@ module TestInterfaces where
 
 import Daml.Script
 import DA.Assert
+import DA.Types()
```

Comment:
was this necessary?

Reply:
Does not seem we need it actually. 
:fire: